Apps/Src/UpdateDisplay.c

```diff
@@ -73,7 +73,7 @@ const char* compStrings[15]= {
 static void assertUpdateDisplayError(UpdateDisplayError_t err){
 	OS_ERR os_err;
 
-	if(err != UPDATEDISPLAY_ERR_NONE){
+	if(err != UPDATEDISPLAY_ERR_NONE || err != UPDATEDISPLAY_ERR_NO_CHANGE){
```

Comment:
Why do we fault on No Change? I thought it just meant that the value did not change since when we last updated it.

Reply:
This is to make sure we don't fault on no change anymore.

Reply:
Makes sense, don't know what i was thinking.